### PR TITLE
ZEPPELIN-992 Move some tests from InterpreterFactoryTest to LazyOpenInterpreterTest

### PR DESCRIPTION
**Pull request by jongyoul**
### What is this PR for?
Moving Interpreter.interpret into LazyOpenInterpreterTest in oder to divide test scope between InterpreterFactoryTest and LazyOpenInterpreter. This is related to #987 a little bit.

### What type of PR is it?
[Refactoring]

### Todos
* [x] - Divide tests

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-992

### How should this be tested?
Changed only test case

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by jongyoul** [comment]
The failures is related to ZEPPELIN-878

**Comment by bzz** [comment]
@jongyoul the CI failure I can see does not seem to be part of any existing open sub-task in [ZEPPELIN-878](https://issues.apache.org/jira/browse/ZEPPELIN-878)

```
09:02:21,815 ERROR org.apache.zeppelin.AbstractZeppelinIT:153 - Exception in ZeppelinIT while testSparkInterpreterDependencyLoading 
org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":"//div[@class='modal-dialog'][contains(.,'Do you want to update this interpreter and restart with new settings?')]//div[@class='modal-footer']//butt
```

**Comment by jongyoul** [comment]
@bzz Thanks for the guide. I'm good at selenium, thus I thought ZeppelinIT.testSparkInterpreterDependencyLoading was related to that issue. I'll look into it but do you know how to reproduce or why it occurs? I re-triggered it and passed all tests.

File: zeppelin-interpreter/src/test/java/org/apache/zeppelin/interpreter/LazyOpenInterpreterTest.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+/**
+ * Created by jl on 6/13/16.
```
Comment (Leemoonsoo) [inline zeppelin-interpreter/src/test/java/org/apache/zeppelin/interpreter/LazyOpenInterpreterTest.java]: It looks like an auto generated comment from some ide.
Could you take care of this comment to not include author?

**Comment by bzz** [comment]
Looks great to me and CI passes now.

**Pull request closed without merging**